assets/src/application/ui/display/espressoIcon/types.ts

```diff
@@ -1,5 +1,5 @@
 export interface EspressoIconProps {
-	icon: Icon | string;
+	icon: Icon | keyof SvgPath;
```

Comment:
SvgPath is basically `[key in Icon]` so is this not redundant and/or using circular references?

Reply:
That means `SvgPath` has its keys (`[key in Icon]`) as `'ticket-alt'`, `'edit` etc., that means you can pass either `Icon.EDIT` or `'edit'` to `EspressoIcon`

Reply:
ahh ok... thnx for the explanation